march_data_collector/src/march_data_collector/cp_calculator.py

```diff
@@ -72,35 +75,41 @@ def _calculate_capture_point(self, duration):
         :param duration:
             the amount of seconds away from the current time the capture point should be calculated
         """
-        falling_time = InvertedPendulum.calculate_falling_time(
-            self._center_of_mass.x,
-            self._center_of_mass.y,
-            self._center_of_mass.z,
-            self.vx, self.vy)
+        try:
+            world_transform = self._tf_buffer.lookup_transform('world', self._static_foot_link, rospy.Time())
+            wt_translation = world_transform.transform.translation
```

Comment:
```suggestion
            static_foot_transform = self._tf_buffer.lookup_transform('world', self._static_foot_link, rospy.Time())
            static_foot_position = world_transform.transform.translation
```
I think something like this is more intuitive